include/gridtools/stencil_composition/backend_cuda/fused_mss_loop_cuda.hpp

```diff
@@ -55,7 +55,10 @@ namespace gridtools {
                     k_caches_type<Mss> k_caches;
                     auto mixed_ptr = hymap::device::merge(k_caches.ptr(), wstd::move(ptr));
                     tuple_util::device::for_each(
-                        [&](int_t size, auto info) {
+                        [&](int_t size, auto info) GT_FORCE_INLINE_LAMBDA {
+#ifdef __clang__
+#pragma unroll 3
```

Comment:
add comment how this value was chosen, to give a pointer to the next developer when it would make sense to change it.